Update docs (#2172)
castorini/anserini: docs/experiments-msmarco-passage.md

As a next step in the onboarding path, you basically [do the same thing again in Python with Pyserini](https://github.com/castorini/pyserini/blob/master/docs/experiments-msmarco-passage.md) (as opposed to Java with Anserini here).

Before you move on, however, add an entry in the "Reproduction Log" at the bottom of this page, following the same format: use `yyyy-mm-dd`, make sure you're using a commit id that's on the main trunk of Anserini, and use its 7-hexadecimal prefix for the link anchor text.

> In the text of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).

Reviewer: ```suggestion
In the description of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).
```

> In the text of your pull request, please provide some details on your setup (for example, operating system, environment and configuration, etc.).

Reviewer: ```suggestion
In the text of your pull request, please provide some details on your setup (e.g., operating system, environment and configuration, etc.).
```

In addition, also provide some indication of success (e.g., everything worked) or document issues you encountered.

> If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include in the pull request.

Reviewer: ```suggestion
If you think this guide can be improved in any way (e.g., you caught a typo or think a clarification is warranted), feel free to include it in the pull request.
```
